README.md

```diff
@@ -14,3 +15,33 @@ In addition, we are trying to have a well-designed structure ourselves so that v
 
 Through these activities, ONE-vscode can provide a differentiation that cannot be experienced only with the existing CLI by linking the toolchain with the execution environment, ONE Runtime, or a target simulator corresponding to the backend compiler. This will eventually lead to fun and high productivity for developers.
 
+## Overview and tutorials
+
+- [ONE-vscode extension overview](./docs/Overview.md)
+- [How to install?](./docs/HowToInstall.md)
+- [How to use?](./docs/HowToUse.md)
+- [Glossary](./docs/Glossary.md)
+
+## Features
+
+<img src="https://user-images.githubusercontent.com/7223627/175224054-0df3109a-f4ef-4a12-a7be-ab50f01b77af.png" width=500 />
+
+- Install toolchain
+
+<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500 />
```

Comment:
This installation gif is good but it's meaningful when user installed npu backend.

One way to show this gif would be
First, we could mention like the following
`"The following image shows an example of installing some toolchain provided by backend provider"` (?) 
and then we could show gif.  :-)

Reply:
Like this?

```suggestion
The following image shows an example of installing some toolchain provided by backend provider
<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500 />
```